scripts/clean.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs')
+const path = require('path')
+const deleteFiles = require('delete')
+
+const root = (...args) => path.join(__dirname, '..', ...args)
+
+const files = fs.readdirSync(root('modules'))
+  .filter(file => file.includes('.js'))
+
+const promises = Promise.all(
+  files.map(file => deleteFiles.promise(root(file)))
```

Comment:
Why not just `fs.unlinkSync()`? `delete` brings in [a lot of excess dependencies](https://github.com/jonschlinkert/delete/blob/master/package.json#L24-L29)